examples/refresh.rb

```diff
@@ -0,0 +1,21 @@
+lib = File.expand_path('../../lib', __FILE__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+
+require 'fritzbox/smarthome'
+
+Fritzbox::Smarthome.configure do |config|
+  config.endpoint   = 'https://fritz.box'
+  config.username   = 'smarthome'
+  config.password   = 'verysmart'
+  config.verify_ssl = false
+end
+
+# Getting a list of actors
+
+pp actors = Fritzbox::Smarthome::Actor.all
+
+# Finding by AIN
+pp actor = Fritzbox::Smarthome::Actor.find_by!(ain: actors.first.ain)
```

Comment:
To illustrate the use case that one doesn't have to fetch the full list first:

```suggestion
pp actor = Fritzbox::Smarthome::Actor.find_by!(ain: "0815-4711") # example AIN
```

Reply:
Yes that would make sense.

I'll check how I will do that and still keep the example run-able for everyone.

Intention here was to not hardcode an AIN as not everyone would have this corresponding device on their FritzBox ;)